rotkehlchen/chain/ethereum/modules/liquity/trove.py

```diff
@@ -84,8 +84,32 @@ def __init__(
     def get_positions(
             self,
             given_addresses: Sequence[ChecksumEvmAddress],
-    ) -> dict[ChecksumEvmAddress, Trove]:
-        """Query liquity contract to detect open troves"""
+    ) -> dict[str, Any]:
+        """Query liquity contract to get the collateral ratio and detect open troves"""
+
+        eth_price = Inquirer().find_usd_price(A_ETH)
+        lusd_price = Inquirer().find_usd_price(A_LUSD)
+
+        try:
+            system_collateral = self.trove_manager_contract.call(
+                node_inquirer=self.ethereum,
+                method_name='getEntireSystemColl',
+                arguments=[],
+            )
+            system_debt = self.trove_manager_contract.call(
+                node_inquirer=self.ethereum,
+                method_name='getEntireSystemDebt',
+                arguments=[],
+            )
+
+            if system_debt != 0:
+                final_collateral_ratio = eth_price * system_collateral / system_debt * lusd_price
+            else:
+                final_collateral_ratio = None
+
+        except RemoteError as e:
+            log.debug(f'Failed {e!s}')
+
```

Comment:
We try to keep try/except statements as short as possible and only around the code that can raise it. Can you add it only around the contract calls. Also is easier to search later if it has a better message

```suggestion

        except RemoteError as e:
            log.error(f'Failed to query liquity contracts for protocol debt: {e}')

```